features/scroll-driven-animations.yml

```diff
@@ -1,5 +1,5 @@
 name: Scroll-driven animations
-description: CSS scroll-driven animations are a type of CSS animations that don't run over time, but are instead driven by the user's scroll position.
+description: The `animation-timeline`, `scroll-timeline`, and `view-timeline` CSS properties advance animations based on the user's scroll position, instead of the time since the document loaded.
```

Comment:
> , instead of the time since the document loaded.

Seems unnecessarily narrow, could be removed.

Reply:
+1, the time since the document loaded doesn't always play a role in time-based animations.

Reply:
Thank you both!